### ISSUE TITLE
Desktop resolution not restored on exit with CRT SwitchRes (Linux)

### ISSUE THREAD

**Issue by atylmo** [libretro/RetroArch#8849]
## Description

When CRT SwitchRes is turned on, exiting RetroArch leaves my resolution at the one used by the last loaded content instead of switching back to what it was before I opened RA.

This also happens on @alphanu1‘s MME4CRT fork, so I wasn’t sure where to file the issue.

### Expected behavior

RetroArch remembers my resolution on startup and restores it on exit.

### Actual behavior

The resolution stays the same as the content that was loaded last. I have to set it again with xrandr.

### Steps to reproduce the bug

1. Turn on CRT SwitchRes
2. Load any content that will switch resolutions
3. Exit RetroArch, either through the menu or the keyboard shortcut
4. Notice that the resolution remains what it was when you exited the content

### Bisect Results

This has happened as long as I’ve been using SwitchRes.

### Version/Commit
You can find this information under Information/System Information

- RetroArch: 1.7.7 / 237e8c3d02

### Environment information

- OS: Ubuntu 19.04
- Compiler: GCC 8.3.0

I don’t know if it helps but I’m using a VGA monitor, so I set it to 31 kHz mode rather than 15kHz.

**Comment by LibretroAdmin** [comment]
@alphanu1 Is this still ongoing or can it be closed?

**Comment by alphanu1** [comment]
Yep this can be closed off. This can still happen if RetroArach crashed before running the exit code. However, this is quite rare. @LibretroAdmin
